pom.xml

```diff
@@ -89,8 +89,8 @@
 				<version>3.3</version>
 				<configuration>
 					<encoding>UTF-8</encoding>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>8</source>
```

Comment:
I wouldn't change the source version. We try to keep the code backwards compatible. IMO there's no reason we might need the features of java 8.